.gitlab/source_test/go_generate_check.yml

```diff
@@ -0,0 +1,17 @@
+---
+# check that go generate has been run in the pkg/security directory
+security_go_generate_check:
+  extends: .tests_linux_ebpf
```

Comment:
I think we don't need this: the `stage` is already set and it adds `artifacts` which we don't want here
```suggestion
```